### ISSUE TITLE
in mnist folder，why teacher_mnist and stdudent_mnist do not contain the softmax?

### ISSUE THREAD

**Issue by parquets** [peterliht/knowledge-distillation-pytorch#27]

**Comment by briliantnugraha** [comment]
Hi @parquets, FYI, they add softmax when they compute the loss function, see kd loss here https://github.com/peterliht/knowledge-distillation-pytorch/blob/master/model/net.py#L110-L112

**Comment by parquets** [comment]
> Hi @parquets, FYI, they add softmax when they compute the loss function, see kd loss here https://github.com/peterliht/knowledge-distillation-pytorch/blob/master/model/net.py#L110-L112

thank you， I get it

**Comment by parquets** [comment]
@peterliht Could you give me pre-trained model for ResNet110?
